### ISSUE TITLE
feat: Call of Duty Black Ops 3

### ISSUE THREAD

**Issue by dgibbs64** [gamedig/node-gamedig#464]
**What is this feature about?**
Add Call of Duty Black Ops 3.

**Additional context/references**
 It has the same query type as Modern Warfare 3

**Comment by podrivo** [comment]
Hey @dgibbs64.

Thank you! We'll make sure to add this as an option in the games list.
In the mean time, you can currently use the Modern Warfare 3 query type to fetch server info:

```js
gamedig --type codmw3 --pretty 108.61.158.93:27017

{
  "name": "Aperture BO3 Classic Svr",
  "map": "mp_stronghold",
  "password": false,
  "raw": {
    "protocol": 17,
    "folder": "usermaps",
    "game": "Black Ops 3",
    "appId": 311210,
    "numplayers": 0,
    "numbots": 0,
    "listentype": "d",
    "environment": "w",
    "secure": 1,
    "version": "1.0.0.0",
    "steamid": "90179601157573656",
    "tags": [
      "\\gametype\\tdm\\dedicated\\true\\ranked\\false\\hardcore\\false\\zombies\\false\\modName\\usermaps\\playerCount\\0"
    ]
  },
  "maxplayers": 18,
  "players": [],
  "bots": [],
  "connect": "108.61.158.93:27017",
  "ping": 0
}
```

**Comment by CosminPerRam** [comment]
Closed by #473.
